WcaOnRails/app/webpacker/javascript/edit-schedule/SchedulesEditor/index.js

```diff
@@ -72,28 +67,37 @@ export default class SchedulesEditor extends React.Component {
           />
         </Col>
         <Col xs={9}>
-          <Panel>
-            <Panel.Heading>
-              <RoomSelector
-                scheduleWcif={scheduleWcif}
-                selectedRoom={selectedRoom}
-                handleRoomChange={handleRoomChange}
-              />
-            </Panel.Heading>
-            <Panel.Body>
-              {selectedRoom.length === 0 ? (
-                <div>Please select a room to edit its schedule</div>
-              ) : (
-                <EditScheduleForRoom
-                  scheduleWcif={scheduleWcif}
-                  locale={locale}
-                  keyboardEnabled={keyboardEnabled}
-                  handleKeyboardChange={handleToggleKeyboardEnabled}
-                  selectedRoom={selectedRoom}
-                />
-              )}
-            </Panel.Body>
-          </Panel>
+          {scheduleWcif === null
+          || scheduleWcif.venues === null
+          || scheduleWcif.venues.length === 0 ? (
```

Comment:
Any specific reason for this detailed check? I believe the venues should always be a list.